tests/scenarios/v2-addon-dev-test.ts

```diff
@@ -73,6 +73,8 @@ appScenarios
               exclude: ['**/-excluded/**/*'],
             }),
 
+            addon.clean(),
```

Comment:
Is this change in order required? If so, then this is clearly a breaking change!

Reply:
I think it is. I need to revalidate

Reply:
Any news on this @patricklx? 

With ~250 addons in our monorepo (not all v2 though), I'd rather not have to write a codemod to change the order! 😏

Reply:
hi @simonihmig I reverted the change and the tests passed. (other unrelated failed)

Reply:
hold that, i forgot one, lets see

Reply:
There was a timeout in one test job. I restarted and it is all green now! :tada: 

Thanks for getting back to this, I think this revert is helpful, as it shows we are not introducing breaking changes here!